documentation/misc/Building_a_network_skeleton.md

```diff
@@ -458,7 +458,7 @@ Typically it's safe to not upgrade filecoin-ffi's version of go-state-types.  Th
 
         config := migration.Config{
             MaxWorkers:        uint(workerCount),
-            ProgressLogPeriod: time.Minute * 5,
+            ProgressLogPeriod: time.Second * 2,
```

Comment:
you'll need to put in your new code here, both the `getMigrationProgressLogTimeout` and the use of `ProgressLogPeriod: timeoutDuration`. Just copy/paste what you have into here and the block below.